xbmc/DatabaseManager.cpp

```diff
@@ -230,3 +230,15 @@ void CDatabaseManager::UpdateStatus(const std::string &name, DB_STATUS status)
   std::unique_lock<CCriticalSection> lock(m_section);
   m_dbStatus[name] = status;
 }
+
+void CDatabaseManager::LocalizationChanged()
+{
+  std::unique_lock<CCriticalSection> lock(m_section);
+
+  // update video version type table after language changed
+  CVideoDatabase videodb;
+  if (videodb.Open())
+    videodb.UpdateVideoVersionTypeTable(false);
+
+  CLog::Log(LOGDEBUG, "{}, Video version type table updated for new language settings", __FUNCTION__);
```

Comment:
Well, I unresolve this review comment, because you addressed only one out of three things I want to discuss so far.

The log is still wrong in case db cannot be opened and my concern re. not silently leaving the method when db cannot be open is still something I want to see changed.

Reply:
By all means log the error but Kodi should continue to run in any case.  It will be obvious in the GUI when all the video stuff disappears that there is an issue with the db.  

My installations use  shared mariadb's but there is always a chance that something on the network goes awry and access is stopped for a time.  I don't think we should be throwing an error as it is recoverable - ie, once the network is back, access is again available.

Reply:
I agree with @the-black-eagle lack of db access can be much like lack of network file access, something temporary awry and recoverable. This is why we do what we do in other places, and what is coded here is fine.

Reply:
> db access can be much like lack of network file access

Maybe, but that we handle this kind of disconnect/reconnect situations correct is just theory. But as two team members say it's fine, so be it.
 
Then, I only request to change here that the line 

`CLog::Log(LOGDEBUG, "{}, Video version type table updated for new language settings", __FUNCTION__);`

should only be called if `videodb.UpdateVideoVersionTypeTable(false);`was actually called which is NOT the case if db cannot be opened.